src/RwaConduit.sol

```diff
@@ -0,0 +1,24 @@
+pragma solidity 0.5.12;
+
+interface GemLike {
+    function transfer(address,uint256) external returns (bool);
+    function balanceOf(address) external returns (uint256);
+}
+
+
+contract RwaConduit {
+    GemLike public gov;
+    GemLike public dai;
+    address public to;
+
+    constructor(address _gov, address _dai, address _to) public {
+        gov = GemLike(_gov);
+        dai = GemLike(_dai);
+        to = _to;
+    }
+
+    function push() public {
```

Comment:
This should maybe be `external` to save memory copying.

Reply:
I don't think this makes a difference unless you have reference types in calldata arguments.